<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWeek13Class</class>
 <widget class="QMainWindow" name="TestWeek13Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestWeek13</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="CarsListWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="ManufacturerLabel">
        <property name="text">
         <string>Manufacturer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ManufacturerLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="ShowCarsButton">
        <property name="text">
         <string>Show cars</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="NumberCarsLabel">
        <property name="text">
         <string>Number of cars</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="NumberCarsValueLabel">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TestWeek13.qrc"/>
 </resources>
 <connections/>
</ui>
